doc/extensibility/IVetoProjectLoad.md

```diff
@@ -3,4 +3,24 @@
 
 **[Item template:](project_item_templates.md)** Project Load Veto extension
 
-TODO
+To veto a project load there is two extension points `IVetoProjectLoad` and `IVetoProjectPreLoad`.
```

Comment:
May be useful to specify what "veto a project load means" - e.g. prevent the project to load